boards/arm/stm32f3_disco/dts_fixup.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2019 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * This file is a temporary workaround for mapping of the generated information
+ * to the current driver definitions.  This will be removed when the drivers
+ * are modified to handle the generated information, or the mapping of
+ * generated data matches the driver definitions.
+ */
+
+/* LSM303DLHC Accelerometer sensor is using LIS2DH driver */
+#define DT_LIS2DH_DEV_NAME		DT_ST_STM32_I2C_V2_40005400_ST_LSM303DLHC_ACCEL_19_LABEL
+#define DT_LIS2DH_I2C_ADDRESS		DT_ST_STM32_I2C_V2_40005400_ST_LSM303DLHC_ACCEL_19_BASE_ADDRESS
+#define DT_LIS2DH_I2C_MASTER_DEV_NAME	DT_ST_STM32_I2C_V2_40005400_ST_LSM303DLHC_ACCEL_19_BUS_NAME
```

Comment:
If you first convert the driver to use DT_<COMPAT>_<INSTANCE> defines as in #12579 , then these fixups can go away.

Reply:
I know Maureen, but I wanted to do it later on.
In case you want it right before then we'll need to wait next week (provided that I will not commit suicide during the weekend...).

I'm already planning with @galak to finish all the pending ST sensor for this kind of change

Reply:
It seems CI is complaining because LIS2Dh name.... So, let's wait next week and I'll incorporate in this PR also the LIS2DH work.

Reply:
> provided that I will not commit suicide during the weekend

Please don't joke about suicide. If you're not joking, please go talk to someone